implementations/rust/ockam/ockam_command/src/tcp/outlet/create.rs

```diff
@@ -142,7 +148,7 @@ pub fn default_from_addr() -> String {
     "/service/outlet".to_string()
 }
 
-pub async fn run_impl(
+pub async fn async_run(
     ctx: Context,
     (opts, cmd): (CommandGlobalOpts, CreateCommand),
 ) -> miette::Result<()> {
```

Comment:
I think most of these functions (the old `run_impl` defined outside of the command struct) are now unused. Clippy doesn't complain because they are public, but we can do the cleanup after landing this PR.